css/site.css

```
@@ -1,3 +1,9 @@
+#map canvas { cursor: crosshair; }
+
+#docs-content .prose h3 > a > code {
+  background-color: #fff;
```

code blocks also add some padding, recommend turning off the padding as well to keep everything lined up:

```suggestion
  background-color: #fff;
  padding: 0;
```